src/components/TransactionItemRow/index.tsx

```diff
@@ -27,6 +29,8 @@ function TransactionItemRow({
     dateColumnSize,
     shouldShowChatBubbleComponent = false,
     onCheckboxPress,
+    containerStyles,
```

Comment:
Why do we need this? It seems like the only time we pass `containerStyles` it's an empty array

Reply:
These are remnants of the now-removed and no longer needed circle used for selecting a row on its right side.